src/main/webapp/app/view/project/admin/users/project-users.component.ts

```diff
@@ -97,4 +100,9 @@ export class ProjectUsersComponent implements OnDestroy {
   trackByFn(index: number, item: Invitation): number {
     return item.id!;
   }
+
+  delete(invitation: Invitation): void {
+    const modalRef = this.modalService.open(ProjectUserInviteDeleteDialogComponent, { backdrop: 'static' });
+    modalRef.componentInstance.invitation = invitation;
```

Comment:
we normally used an `updateForm` method for modals. but I think it's ok for a single attribute.

Reply:
ja aber hier gibts ja gar keine form. das ist nur ein feld

Reply:
eben, darum ok 👍🏼